src/models/intent.py

```
@@ -1,5 +1,5 @@
 # src/models/schemas.py
-from pydantic import BaseModel, Field, validator
+from pydantic import BaseModel, Field, field_validator, validator
```

Flake8 (7.2.0)

[error] 'pydantic.validator' imported but unused

(F401)